cmd/kubeadm/app/cmd/config_test.go

```diff
@@ -219,80 +240,80 @@ func TestConfigImagesListOutput(t *testing.T) {
 		{
 			name: "text output",
 			cfg: kubeadmapiv1beta2.ClusterConfiguration{
-				KubernetesVersion: dummyKubernetesVersion,
+				KubernetesVersion: dummyKubernetesVersionStr,
 			},
 			outputFormat: "text",
-			expectedOutput: `k8s.gcr.io/kube-apiserver:v1.16.0
-k8s.gcr.io/kube-controller-manager:v1.16.0
-k8s.gcr.io/kube-scheduler:v1.16.0
-k8s.gcr.io/kube-proxy:v1.16.0
-k8s.gcr.io/pause:3.2
-k8s.gcr.io/etcd:3.3.17-0
-k8s.gcr.io/coredns:1.6.5
+			expectedOutput: `k8s.gcr.io/kube-apiserver:{{.KubeVersion}}
+k8s.gcr.io/kube-controller-manager:{{.KubeVersion}}
+k8s.gcr.io/kube-scheduler:{{.KubeVersion}}
+k8s.gcr.io/kube-proxy:{{.KubeVersion}}
+k8s.gcr.io/pause:{{.PauseVersion}}
+k8s.gcr.io/etcd:{{.EtcdVersion}}
+k8s.gcr.io/coredns:{{.CoreDNSVersion}}
```

Comment:
/cc @bart0sh 
i've unpinned the version in the expectedOutput.
pinning them is bad because it causes huge diffs when we bump the underlying version constants in constants.go...